pkg/roachpb/BUILD.bazel

```diff
@@ -53,6 +54,53 @@ go_library(
     ],
 )
 
+# Creating bootstrap library as we need to partition
+# the dependencies here to avoid cyclical structures.
+go_library(
+    name = "bootstrap",
+    srcs = [
+        "api.go",
+        "app_stats.go",
+        "batch.go",
+        "batch_generated.go",
+        "data.go",
+        "errors.go",
+        "metadata.go",
+        "metadata_replicas.go",
+        "method.go",
+        "version.go",
+    ],
+    embed = [":roachpb_go_proto"],
+    importpath = "github.com/cockroachdb/cockroach/pkg/roachpb",
+    visibility = ["//visibility:public"],
+    deps = [
+        "//pkg/geo",
+        "//pkg/geo/geopb",
+        "//pkg/kv/kvserver/concurrency/lock",
+        "//pkg/storage/enginepb",
+        "//pkg/util",
+        "//pkg/util/bitarray",
+        "//pkg/util/caller",
+        "//pkg/util/duration",
+        "//pkg/util/encoding",
+        "//pkg/util/hlc",
+        "//pkg/util/humanizeutil",
+        "//pkg/util/interval",
+        "//pkg/util/log",
+        "//pkg/util/protoutil",
+        "//pkg/util/timetz",
+        "//pkg/util/uuid",
+        "@com_github_aws_aws_sdk_go//aws",
+        "@com_github_aws_aws_sdk_go//aws/credentials",
+        "@com_github_cockroachdb_apd_v2//:apd",
+        "@com_github_cockroachdb_errors//:errors",
+        "@com_github_cockroachdb_errors//errorspb",
+        "@com_github_cockroachdb_redact//:redact",
+        "@io_etcd_go_etcd_raft_v3//raftpb",
+    ],
+)
+
+# keep
```

Comment:
Same with above. The other instances of `# keep` in this file should be removed unless we have a really, really good reason.